### DISCUSSION TITLE
feature request of SmoothGraphics plugin - InnerStroke

### DISCUSSION THREAD

**Discussion by humbleCoder1990** [pixijs/pixijs#7698] [Q&A]
SmoothGraphics & Graphics both use 'center stroke' width.
So, If I increase lineWidth, the size of shape is getting bigger (lineWidth / 2)
To draw inner stroke, I can caculate it myself,
But it will be really hand the plugin support it like..

`smoothGraphics.isInnerStroke = true`

Thanks PIXI team always.
It's best tool I use everyday..

**Comment by bigtimebuddy** [comment]
We have a stroke style property called align which does this. The default is 0.5. A value of 0 is inside and 1 is outside.

**Reply by humbleCoder1990** [reply]
Thanks Matt. 
It's an amazing plug-in.
Please add it as built-in to Pixi, not plug-in.
